### DISCUSSION TITLE
Laptop speakers guide doesn't mention (2L) or (3L) for "Oversampling to Half x4" which is missing

### DISCUSSION THREAD

**Discussion by pinbuck** [wwmm/easyeffects#3555] [Q&A]
https://wwmm.github.io/easyeffects/guide_1.html
"Oversampling" for the Limiter section doesn't mention (2L) or (3L), or which to choose within the guide. In the "Limiter" category of the wiki, it says "kernels" or something, but has no reference of what that means, nor could I find it online.

What would be the better option to choose, and why? It would be great if that was also mentioned in the guide.
![image](https://github.com/user-attachments/assets/57f69fd9-b859-43aa-80cc-217225204265)

**Comment by wwmm** [comment]
Our manuals need some updates indeed. For the limiter you can read more technical information in its author page https://lsp-plug.in/?page=manuals&section=sc_limiter_stereo.

Personally I never felt the need to do oversampling in the limiter. As the plugins we use have audio engineers and musicians as primary target I imagine that in a professional work it may be necessary to use more advanced modes of operation.

**Comment by pinbuck** [comment]
Thank you 👍
